chrome/manifest.json

```diff
@@ -9,6 +9,13 @@
   "page_action": {
     "default_popup": "views/popup.html"
   },
-  "permissions": ["declarativeContent"],
+  "permissions": ["declarativeContent", "activeTab"],
+  "content_scripts": [
+    {
+      "matches": ["http://*/*","https://*/*"],
```

Comment:
This is to have the content script run on as many sites as possible. If we ever decide to extend to files we have to add that type here